src/syft/core/io/route.py

```diff
@@ -3,24 +3,54 @@
 from typing import final
 
 
+class BaseRoute(object):
+    connection_details = {}
+
+    def configure_connection(self, protocol: str, host: str, port: int):
+        """
+        the route should have connection details embedded in it.
+        so that nodes operators can utilize it to route messages.
+        """
+        self.connection_details.update({
+            'host': host,
```

Comment:
I think about it slightly differently - but not with high confidence. My assumption is that the route should contain "Send me to X node" but that the worker's job is to maintain its own connection and protocol to that node. 

Alternatively, we could consider these details to be optional - things that the node could choose to ignore (or if this protocol wasn't available, it could try to get to the destination through another means).

Or we could consider these to be gold standard and assume that the node would do the heavy lifting to create this kind of connection if it didn't already exist.

All are interesting options.